styles.css

```diff
@@ -777,6 +777,189 @@ a {
   color: #f55800;
 }
 
+/* Mobile popup section */
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: fixed;
+  left: 0;
+
+  /* right: 0; */
```

Comment:
Please don't leave any uncommented code in your codebase. In this way, [you can keep your codebase clean, organized, and professional](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.). 🫧